### PR TITLE
Add ced metadata

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
@bendichter @luiztauffer @alejoe91 Finished adding the other data fields in the example .smrx files.

Features Alessio's algorithm for extracting start/stop interval points from the TTL signals for each stimulus.

### CODE REVIEW

File: mease_lab_to_nwb/convert_ced/cedstimulusinterface.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            description="Activation times inferred from TTL commands for mechanical stimulus."
+        )
+        laser_stim = TimeIntervals(
+            name='LaserStimulus',
+            description="Activation times inferred from TTL commands for cortical laser stimulus."
+        )
+        for j, table in enumerate([mech_stim, laser_stim]):
+            for row in conditions[j]:
+                table.add_row(dict(start_time=row[0], stop_time=row[1]))
+        check_module(nwbfile, 'behavior', "Contains behavioral data.").add_data_interface(mech_stim)
```
Comment (bendichter) [inline mease_lab_to_nwb/convert_ced/cedstimulusinterface.py:104]:
```suggestion
        check_module(nwbfile, 'behavior', "Contains behavioral data.").add(mech_stim)
```

File: mease_lab_to_nwb/convert_ced/cedstimulusinterface.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        check_module(nwbfile, 'behavior', "Contains behavioral data.").add_data_interface(laser_stim)
```
Comment (bendichter) [inline mease_lab_to_nwb/convert_ced/cedstimulusinterface.py:105]:
```suggestion
        check_module(nwbfile, 'behavior', "Contains behavioral data.").add(laser_stim)
```

File: mease_lab_to_nwb/convert_ced/convert_ced.py [hunk 3]
```diff
@@ -8,19 +11,44 @@
 ced_file_path = base_path / "m365_pt1_590-1190secs-001.smrx"
 nwbfile_path = base_path / "CED_stub.nwb"
 
+# Enter Session and Subject information here
+session_description = "Enter session description here."
+
+# Manually insert the session start time
+session_start = datetime(1971, 1, 1, 1, 1, 1)  # (Year, Month, Day, Hour, Minute, Second)
```
Comment (bendichter) [inline mease_lab_to_nwb/convert_ced/convert_ced.py:18]:
```suggestion
session_start = datetime(1970, 1, 1)  # (Year, Month, Day)
```

**Review by bendichter — CHANGES_REQUESTED** [review]

File: mease_lab_to_nwb/convert_ced/convert_ced.py (hunk 3 above)
Comment (CodyCBakerPhD) [reply mease_lab_to_nwb/convert_ced/convert_ced.py:18]: There appears to be a known windows specific issue where `datetime(1970,1,1)` can't be used with `.astimezone()`, so I incremented it to 1971. Should I just remove the `astimezone` instead?

**Pull request merged**